t/APISIX.pm

```diff
@@ -260,6 +261,17 @@ _EOC_
         require("apisix").http_init_worker()
     }
 
+    # only for test
```

Comment:
you can add new function in `t/lib/server.lua`, it is enough for testing

Reply:
we need to delete those conf in `t/APISIX.PM`

Reply:
OK. I'll update it.